### PR TITLE
Implement No Conflict Update and Merge

### PR DESCRIPTION
**Pull request by mandolyte**
# Describe what your pull request addresses

- [ ] This allows an admin to update a user branch from master or to merge a user branch into master.

## Test Instructions

- [ ] Login, select org and language where you have access
- [ ] On nav drawer, select the "Update/Merge Changes" Button
- [ ] The status line at the bottom of the page will say "Select a resource"
- [ ] Follow the advice given on the status line
- [ ] Confirm in DCS the changes at every step

### CODE REVIEW

**Comment by richmahn** [comment]
@mandolyte Found a bug. If you use server=QA and then login and pick dcs-poc-org and en, and then Translation Notes, and then branch-ahead, it is right, but then if you go to branch-behind, it doesn't disable the merge to master button as it should:

image

It does have it disabled if you go directly to branch-behind without checking branch-ahead first.

File: pages/branchMerge.js [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+                  name="resource-type-radio-buttons-group"
+                  row
+                  onChange={handleResourceChange}
+                >
+                  {resources.map( ({ id,name }) =>
+                    <FormControlLabel key={id} value={id} control={<Radio />} label={name} />,
+                  )}
+                </RadioGroup>
+              </FormControl>
+              <Autocomplete
```
Comment (richmahn) [inline pages/branchMerge.js:265]: Can you have this say "Select a Branch"?

image

**Review by richmahn — CHANGES_REQUESTED** [review]

File: pages/branchMerge.js (hunk 1 above)
Comment (richmahn) [reply pages/branchMerge.js:265]: Also, are we going to tell them this branch has conflicts, can't do anything, or just that there are no changes?

Comment (mandolyte) [reply pages/branchMerge.js:265]: Both comments are resolved with most recent pushes.

**Review by richmahn — APPROVED** [review]
LGTM

**Pull request merged**